### PR TITLE
Microsoft graph api bug fix 2

### PR DESCRIPTION
**Pull request by Shellyber**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/47842

## Description
- Fixed an issue in which the **msgraph-api-auth-start** command returned an error for Cortex XSOAR azure app.
- Added more information to the detailed description regarding self deployed app

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI_description.md [hunk 1]
```diff
@@ -26,4 +26,6 @@ For more information, refer to the following [article](https://xsoar.pan.dev/doc
 
 The ***Application Secret*** and the ***Tenant ID*** integration parameters are required for this method.
 
+Run the *msgraph-api-test* command to ensure connectivity to Microsoft.
```
Comment (ShahafBenYakir) [inline Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI_description.md:29]:
```suggestion
Run the ***msgraph-api-test*** command to ensure connectivity to Microsoft.
```

File: Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py [hunk 2]
```diff
@@ -141,8 +141,6 @@ def main() -> None:  # pragma: no cover
     app_secret = params.get('app_secret') or (params.get('credentials') or {}).get('password')
     certificate_thumbprint = params.get('certificate_thumbprint')
     private_key = params.get('private_key')
-    if not app_secret and not (certificate_thumbprint and private_key):
-        raise DemistoException('Application Secret or Certificate Thumbprint and Private Key must be provided.')
```
Comment (ShahafBenYakir) [inline Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:145]: In the [docs](https://github.com/demisto/content/pull/18493/files#diff-cc1ccabe6d29bfa8c195bc09c669eb91fbb5829e27aca6c4e43eba12c19532d9R27) we are saying that for self-deployed apps ***Application Secret*** and the ***Tenant ID*** should be filled so why are we removing this validation?

Comment (Shellyber) [reply Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:145]: For the self deployed, yes. 
TenentID and app secret. But this error jumps when customers are using the XSOAR app. 
The exception is wrong.

Comment (ShahafBenYakir) [reply Packs/MicrosoftGraphAPI/Integrations/MicrosoftGraphAPI/MicrosoftGraphAPI.py:145]: I see, let's just verify with @DeanArbel who enhanced this part only a month ago

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**